packages/fuels-programs/src/contract.rs

```diff
@@ -1,169 +1,65 @@
-mod load;
 mod storage;
 
-use std::{
-    fmt::Debug,
-    fs,
-    path::{Path, PathBuf},
-};
+use std::fmt::Debug;
 
 use fuel_tx::{Bytes32, Contract as FuelContract, ContractId, Salt, StorageSlot};
-use fuels_accounts::Account;
-use fuels_core::types::{
-    bech32::Bech32ContractId,
-    errors::{error, Result},
-    transaction::TxPolicies,
-    transaction_builders::CreateTransactionBuilder,
-};
-pub use load::*;
 pub use storage::*;
 
-/// [`Contract`] is a struct to interface with a contract. That includes things such as
-/// compiling, deploying, and running transactions against a contract.
-#[derive(Debug, Clone)]
-pub struct Contract {
-    binary: Vec<u8>,
+#[derive(Debug, Clone, PartialEq)]
+pub struct Contract<Code> {
```

Comment:
Nit: having a docstring here would be nice

Reply:
The previous one was mostly false :D 

I'll add one now.